File: GumiPresso_Server/src/main/java/com/ssafy/through/model/service/OrderService.java
package com.ssafy.through.model.service;

import java.util.List;

import com.ssafy.through.model.dto.Order;
import com.ssafy.through.model.dto.Table;

public interface OrderService {

	int insert(Order order);
	List<Order> select(String user_id);
	int search();
	List<Table> getTable();
	int updateTable(int tableId);
}

File: GumiPresso_User/app/src/main/java/com/ssafy/gumipresso/model/service/OrderService.kt
package com.ssafy.gumipresso.model.service

import com.ssafy.gumipresso.model.dto.OrderForm
import com.ssafy.gumipresso.model.dto.RecentOrder
import retrofit2.Response
import retrofit2.http.Body
import retrofit2.http.GET
import retrofit2.http.POST
import retrofit2.http.Path

interface OrderService {
    @GET("/order/recent/{userId}")
    suspend fun getOrderList(@Path("userId")user_id: String) : Response<List<RecentOrder>>

    @GET("/order/order/{orderId}")
    suspend fun getOrderDetailList(@Path("orderId") orderId: String): Response<RecentOrder>

    @POST("/order/new")
    suspend fun order(@Body orderForm: OrderForm): Response<Unit>
}